src/components/mapSearch.js

```diff
@@ -3,6 +3,11 @@ import {APIProvider, Marker, AdvancedMarker, Map} from '@vis.gl/react-google-map
 import GooglePlacesAutocomplete, { geocodeByPlaceId, getLatLng} from 'react-google-places-autocomplete';
 import TextForm from "./textForm";
 
+// This component is the depreciated version of the map search bar, it is not currently used in the project
+// Left for reference in case Jasper wants to use this react google maps package instead of the one used in the project
```

Comment:
Thanks ;)

Reply:
For sure, it took a long time to get that specific react implementation of the google maps working, so if u end up swapping over to that later at least you'll have that much